.babelrc.js

```diff
@@ -23,7 +23,12 @@ module.exports = (api) => {
 					corejs: 3,
 				},
 			],
-			'@babel/preset-react',
+			[
```

Comment:
Why not build with SWC as well and get rid of this?

Reply:
wel we had some serious issues with Jest but I will change it to see what happens